draft-packages/skirt/KaizenDraft/Skirt/SkirtCard.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react"
+import { Card, CardProps } from "@kaizen/draft-card"
+import classNames from "classnames"
+import styles from "./SkirtCard.scss"
+
+export const SkirtCard = (props: CardProps) => {
```

Comment:
Would love to hear your thoughts on a thin wrapper of card to create SkirtCard. Is it to encourage usage? Reinforce pattern?

Reply:
It’s a little of pattern-reinforcement/easy-usage, yep, since the `SkirtCard` is something that’s tied fairly closely to the `Skirt` in design-terms.

The primary reason though is that there’s some design subtlety for the `SkirtCard` at smaller sizes that’s applied here and if we were leaving it up to consumers to implement we’d need to make sure they implement the same details.

You can see in `SkirtCard.scss` that as the screen collapses in size the card gets negative margins so it touches the full width of the screen, and it has slightly border-radius variations.